src/standalone/Bucket/bucket_parameterizations.jl

```diff
@@ -29,12 +29,18 @@ function ClimaLand.surface_temperature(model::BucketModel, Y, p, t)
 end
 
 """
-    ClimaLand.surface_specific_humidity(model::BucketModel, Y, p)
+    ClimaLand.surface_specific_humidity(atmos, model::BucketModel, Y, p)
 
 a helper function which returns the surface specific humidity for the bucket
 model, which is stored in the aux state.
 """
-function ClimaLand.surface_specific_humidity(model::BucketModel, Y, p, _...)
+function ClimaLand.surface_specific_humidity(
```

Comment:
Did this function need to change?

Reply:
I had to change the `surface_specific_humidity` interface to take in `atmos` so we can dispatch on it for the snow case. Since the default `turbulent_fluxes!` in drivers.jl calls this function, I had to change it for the bucket too